readme/Rules.scalatex

```diff
@@ -27,27 +27,28 @@
 
     @ul
       @li
-        @note. This rewrite inserts fully qualified names, even if the short name
+        @note. This rule inserts fully qualified names, even if the short name
         is available in scope. Track @issue(169) for updates on making the
-        rewrite insert shorter names.
+        rule insert shorter names.
       @li
         Inserts type annotations only for implicit definitions by default. To configure the
-        rewrite to run on non-implicit definitions
+        rule to run on non-implicit definitions
         @config
           explicitReturnTypes.memberKind = [Val, Def, Var]
           explicitReturnTypes.memberVisibility = [Public, Protected]
           // Experimental, shorten fully qualified names and insert missing imports
           // By default, names are fully qualified and prefixed with _root_.
           unsafeShortenNames = true // false by default.
-
+  @sect{ExplicitReturnTypes}
+    This rewrite has been renamed into @sect.ref{ExplicitResultTypes}
```

Comment:
s/rewrite/rule

Reply:
Old habit, wrote it after running sed replace. Fixed.